pkg/server/chaosd/attack.go

```diff
@@ -59,29 +66,11 @@ func (s *Server) ExecuteAttack(attackType AttackType, options core.AttackConfig,
 		RecoverCommand: options.RecoverData(),
 		LaunchMode:     launchMode,
 	}
-	if err = s.exp.Set(context.Background(), exp); err != nil {
+	if err = s.expStore.Set(context.Background(), exp); err != nil {
 		err = perr.WithStack(err)
 		return
 	}
 
-	defer func() {
```

Comment:
why remove this defer function?

Reply:
I think  defer function here is not the common use case and it will break the serial code in read order.

Reply:
Fixed.